next/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "next",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start"
```

Comment:
Newb question but what's the difference between dev and start run scripts in nextjs?

Reply:
`dev` is for the local development environment and `start` is for running a production version of the site!